aws/logs_monitoring/tools/publish_layers.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+# Publish the datadog python lambda layer across regions, using the AWS CLI
+# Usage: VERSION=5 REGIONS=us-east-1 LAYERS=Datadog-Python27 publish_layers.sh
+# VERSION is required.
+set -e
+
```

Comment:
It looks like this script requires `VERSION` and `FORWARDER_VERSION`. The Forwarder version seems like a semantic version number whereas version is the layer version. However, the usage of these two terms isn't consistent across all of the scripts.

Could we switch to using `LAYER_VERSION` and `FORWARDER_VERSION` everywhere, so it is easy to keep straight which is which in all of the scripts?